Dockerfile

```diff
@@ -10,38 +10,38 @@ COPY ./node-template/customSpec.json ./runtime/tools/spec-builder/customSpec.jso
 
 WORKDIR /usr/src/runtime
 
-RUN yarn install
-
-RUN yarn run build
-
-RUN mv wasm-code ./tools/spec-builder
+RUN yarn install && yarn run build && mv wasm-code ./tools/spec-builder
 
 WORKDIR /usr/src/runtime/tools/spec-builder
 
-RUN apt-get update -y 
-RUN apt-get install jq -y
-RUN yarn run asbuild && bash cr-custom-spec.sh
-RUN yarn build-spec -f customSpec.json
+RUN apt-get update -y && apt-get install jq -y
+RUN yarn run asbuild && bash cr-custom-spec.sh && yarn build-spec -f customSpec.json
 CMD ["/usr/bin/echo", "succesfully build as-runtime and generated chainspec file"]
 
 # Building node and running it with the customSpecRaw.json
 
 FROM rust:1.45 AS node-builder
 
-WORKDIR /usr/src/dummy
+WORKDIR /usr/src/node
 
 COPY ./node-template ./
 
 COPY --from=builder /usr/src/runtime/tools/spec-builder/customSpecRaw.json ./
 
 RUN bash ./scripts/init.sh
-RUN apt-get update -y 
-RUN apt-get -y install clang
-RUN apt-get -y install gcc
-RUN apt-get -y install cmake
+RUN apt-get update -y &&\
```

Comment:
instead of doing `install` for every dep, you can do `ap-get -y install clang gcc cmake`